README.md

```diff
@@ -126,7 +126,7 @@ $ sudo chmod 660 /dev/igb_avb
 $ sudo chgrp ias_avb /dev/igb_avb
 $ sudo chmod 660 /dev/ptp*
 $ sudo chgrp ias_avb /dev/ptp*
-
+$ sudo chgrp ias_avb /dev/shm/igb_sem
```

Comment:
I'd add a space between this block and the `# Start the DLT daemon...` below. Anyway, I don't have this directory after loading `igb_avb`, but after running `avb_streamhandler_demo` itself, so I believe this instruction belongs to "after the instruction to run avb_streamhandler_demo", otherwise it will fail with `No such file or directory`.

Reply:
Thanks for pointing it out. will update it.